Source/Core/Core/AchievementManager.cpp

```diff
@@ -1175,7 +1251,12 @@ void AchievementManager::HandleLeaderboardTriggeredEvent(const rc_runtime_event_
                                     m_game_data.leaderboards[ix].title),
                         OSD::Duration::VERY_LONG, OSD::Color::YELLOW);
       }
-      return;
+      m_queue.EmplaceItem([this, runtime_event] {
+        FetchBoardInfo(runtime_event->id);
+        if (m_update_callback)
+          m_update_callback();
+      });
+      break;
```

Comment:
This goes for this entire function and I'm not sure why I didn't see this before, but you can't legally access `runtime_event` anymore once the outer function returns. Please copy out the `id` and `value` into local variables and copy them into the lambdas.

Reply:
The `m_queue.EmplaceItem([this, runtime_event] { AwardAchievement(runtime_event->id); });` line in `HandleAchievementTriggeredEvent()` has the same problem.

Reply:
-> #12228

Reply:
This is no longer valid, please rebase.